core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>ArduPilot Manager</title>
+    <link rel="stylesheet" href="/static/css/metro-all.css" />
+    <link rel="stylesheet" href="/style.css" />
+    <script src="/static/js/vue.js"></script>
+    <script src="/static/js/axios.min.js"></script>
+    <script src="/static/js/metro.js"></script>
+</head>
+
+<body>
+    <div id="app">
+        <div class="container">
+            <h1>ArduPilot Manager</h1>
+            <h3>Endpoints:</h3>
+
+            <div class="grid">
+                <div class="row p-1 m-1 bg-gray">
+                    <div class="cell-2">
+                        <div>Name</div>
+                    </div>
+                    <div class="cell-1">
+                        <div>Owner</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-end">Type</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-center">Place</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-start">Argument</div>
+                    </div>
+                    <div class="cell-1">
+                        <span class="mif-floppy-disk mif-lg d-flex flex-justify-center pr-2"></span>
+                    </div>
+                    <div class="cell-1">
+                        <span class="mif-lock mif-lg d-flex flex-justify-center pr-2"></span>
```

Comment:
Why the lock icon does not accept user input ?

Reply:
Currently only the system itself can create protected endpoints.

Protected endpoints created by user's would require some kind of logic about who can and who cannot delete them.

Not saying it can't be done, but if you find there's a better approach for that I would like to discuss it better for a following PR.

Reply:
Well, I'm ok to have it for our development frontend (this one) but not for the final user interface

Reply:
Opened #230 to better discuss that. For this PR I think we should ignore the discussion for the time, as it implements the frontend behavior according to current's backend behavior.